docs/guides/setup-channel.mdx

```diff
@@ -121,20 +118,61 @@ print(result)
   ```
 
 <span id="iqp-rest-api"></span>
-## Set up to use IBM Quantum Platform with REST API
+### Set up to use IBM Quantum Platform with REST API
 
 Alternatively, you can also access quantum processors with REST APIs, enabling you to work with QPUs using any programming language or framework. 
 
-1. Retrieve your IBM Quantum token from the [IBM Quantum account page](https://quantum.ibm.com/account).
-
-2. If you do not already have a user account, get one at the [IBM Quantum login page.](https://quantum.ibm.com/login)  Your user account is associated with one or more [instances](./instances) (in the form hub / group / project) that give access to IBM Quantum services. Additionally, a unique token is assigned to each account, allowing for IBM Quantum access from Qiskit. The instructions in this section use our default instance.  For instructions to choose a specific instance, see [Connect to an instance](./instances#connect-instance).
+1. If you do not already have a user account, get one at the [IBM Quantum login page.](https://quantum.ibm.com/login)  Your user account is associated with one or more [instances](./instances) (in the form hub / group / project) that give access to IBM Quantum services. Additionally, a unique token is assigned to each account, allowing for IBM Quantum access from Qiskit. The instructions in this section use our default instance.  For instructions to choose a specific instance, see [Connect to an instance](./instances#connect-instance).
 
     <Admonition type="note">
       The Instances section in your [IBM Quantum account page](https://quantum.ibm.com/account) lists the instances that you can access.
     </Admonition>
 
+1. Retrieve your API token from the [IBM Quantum account page](https://quantum.ibm.com/account).
+
+1. Because of the security risks posed by executing code that contains your API token, the recommended authentication method is to first create an environment variable for your API token, but only do so **if you are working in a trusted Python environment (such as on a personal laptop or workstation).** [Skip to the next step](#rest-untrusted) if you are not using a trusted environment, such as a shared or public computer.
+
+    To set the IQP_API_TOKEN environment variable in your system, you can add the following line to your shell profile (for example, .bashrc or .zshrc) or by setting it directly in your terminal:
+
+    ```shell
+    export IQP_API_TOKEN=<your-token>
+    ```
+    When you invoke the environment variable in your code, include `import os`, as in this example request:
+
+    ```python
+    import os
+    import requests
+    
+    reqUrl = "https://api.quantum-computing.ibm.com/runtime/jobs?limit=10&offset=0&exclude_params=true"
+    
+    headersList = {
+      "Accept": "application/json",
+      "Authorization": "Bearer os.environ.pop(IQP_API_TOKEN)"
+    }
```

Comment:
The code to get the token from env variable needs to actually run, e.g. 

```suggestion
    headersList = {
      "Accept": "application/json",
      "Authorization": f"Bearer {os.environ['IQP_API_TOKEN']}"
    }
```

Also don't use `pop()` since it removes the token from the env variable.